projects/element-ng/chat-messages/si-attachment-list.component.ts

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) Siemens 2016 - 2025
+ * SPDX-License-Identifier: MIT
+ */
+import { booleanAttribute, Component, inject, input, output, TemplateRef } from '@angular/core';
+import { SiIconComponent } from '@siemens/element-ng/icon';
+import { SiModalService } from '@siemens/element-ng/modal';
+import { SiTranslatePipe, t } from '@siemens/element-translate-ng/translate';
+
+export interface AttachmentItem {
```

Comment:
I would just call it 


```suggestion
export interface Attachment {
```